CMakeLists.txt

```diff
@@ -0,0 +1,256 @@
+cmake_minimum_required(VERSION 3.15.0)
+project(mapnik 
+    VERSION 3.0.24 
```

Comment:
We should target version 4.x.x (master) Exact version can be extracted from `mapnik/version.hpp`
@mathisloge

Reply:
in normal cmake projects one will set the version in the cmake file and generating the mapnik/version.hpp with cmake itself. 
https://cmake.org/cmake/help/latest/guide/tutorial/index.html#adding-a-version-number-and-configured-header-file

but since we need to maintain both build systems(?) SCons and cmake i will look into a way to first extract the version of mapnik/version.hpp and writing the version to the project definition.

Reply:
done with 5f02b19